components/telegram/sources/channel-updates/channel-updates.js

```diff
@@ -0,0 +1,45 @@
+const telegram = require("../../telegram.app.js");
+
+module.exports = {
+  key: "telegram-channel-updates",
+  name: "Channel Updates (Instant)",
+  description: "Emits an event each time a channel message is created or updated.",
+  version: "0.0.1",
+  props: {
+    db: "$.service.db",
+    http: {
+      type: "$.interface.http",
+      customResponse: true,
+    },
+    telegram,
+  },
+  hooks: {
+    async activate() {
+      const response = await this.telegram.createHook(this.http.endpoint, ['channel_post', 'edited_channel_post']);
```

Comment:
need to save off the id you get back here

Reply:
also remove the `console.log()` calls in the activate/deactivate hooks

Reply:
No id is returned here. I had accidentally left hookId as a param in deleteHook(), but have now removed it since it is not needed.